### PR TITLE
Refactor CLI test mixin

### PR DESCRIPTION
**Pull request by xbrianh**
This will be useful for a reorg of tests into separate modules.

depends on #172 #173

### CODE REVIEW

File: tests/__init__.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+import argparse
+import subprocess
+from contextlib import redirect_stdout
+from tempfile import NamedTemporaryFile
+from typing import List, Callable
+
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))  # noqa
+sys.path.insert(0, pkg_root)  # noqa
+
+from tests import config
```
Comment (DailyDreaming) [inline tests/__init__.py:13]: Unused import?

**Review by DailyDreaming — APPROVED** [review]
LGTM.  See comment.

File: tests/__init__.py (hunk 1 above)
Comment (xbrianh) [reply tests/__init__.py:13]: Nope, it configures the test environment.

Comment (xbrianh) [reply tests/__init__.py:13]: I'll add a comment.

**Pull request merged**